code/game/area/areas/ruins/space.dm

```diff
@@ -699,3 +699,49 @@
 /area/ruin/space/has_grav/garbagetruck/medicalwaste
 /area/ruin/space/has_grav/garbagetruck/squat
 /area/ruin/space/has_grav/garbagetruck/toystore
+
+//Donk Co trading outpost
+/area/ruin/space/has_grav/hauntedtradingpost
+	name = "\improper Donk Co. Trading Outpost"
+	icon_state = "donk_public"
+	sound_environment = SOUND_AREA_STANDARD_STATION
+/area/ruin/space/has_grav/hauntedtradingpost/public
+	name = "\improper Donk Co. Trading Outpost Public Meeting Area And Cafeteria"
+/area/ruin/space/has_grav/hauntedtradingpost/public/corridor
+	name = "\improper Donk Co. Trading Outpost Public Docks And Corridors"
+	sound_environment = SOUND_AREA_LARGE_ENCLOSED
+/area/ruin/space/has_grav/hauntedtradingpost/employees
+	name = "\improper Donk Co. Trading Outpost Employees Lounge"
+	icon_state = "donk_employees"
+	airlock_wires = /datum/wires/airlock/engineering
+	sound_environment = SOUND_AREA_MEDIUM_SOFTFLOOR
+/area/ruin/space/has_grav/hauntedtradingpost/employees/workstation
+	name = "\improper Donk Co. Trading Outpost Engineering Station"
+/area/ruin/space/has_grav/hauntedtradingpost/employees/corridor
+	name = "\improper Donk Co. Trading Outpost Secure Corridor"
+	icon_state = "donk_command"
+/area/ruin/space/has_grav/hauntedtradingpost/employees/breakroom
+	name = "\improper Donk Co. Trading Outpost Break Room"
+/area/ruin/space/has_grav/hauntedtradingpost/maint
+	name = "\improper Donk Co. Trading Outpost Auxiliary Storage Room"
+	icon_state = "donk_maints"
+	airlock_wires = /datum/wires/airlock/maint
+	sound_environment = SOUND_AREA_TUNNEL_ENCLOSED
+	ambience_index = AMBIENCE_MAINT
+/area/ruin/space/has_grav/hauntedtradingpost/maint/toolstorage
+/area/ruin/space/has_grav/hauntedtradingpost/maint/toystorage
+/area/ruin/space/has_grav/hauntedtradingpost/maint/disposals
```

Comment:
t-t-triple ditto, Also these  are missing their own names.

Reply:
Sharing their name is intended. Only reason they're different areas is so each can get an APC. Although looking at it again, I'll slap a unique name on /disposals since it's not really a 'storage room'.

Reply:
This is going to confuse other mappers in the future.